OpenTabletDriver.Web/Views/Wiki/FAQ/Windows.cshtml

```diff
@@ -2,70 +2,112 @@
+        joining our support <a href="/Discord">Discord</a> for further help.<br/>
+
+    <h5>Invalid WinUSB driver installation state.</h5>
+        <codeblock class="mt-2", language="sh">
+            Failed to open file handle to WinUSB interface
+            SafeHandle cannot be null. (Parameter 'pHandle')
+        </codeblock>
+        This is usually resolved when you restart your PC.<br/><br/>
+
+        If nothing above applies, remove <code>%localappdata%/OpenTabletDriver</code> or the contents of the <code>userdata</code> folder next to the binaries (if applicable).<br/>
+        If you still need more help after this, join our Discord and give us a shout in the <code>#support-windows</code> channel, mentioning what you have tried.
+    </p>
+
+    <h4>It is not a supported tablet, what can I do about that?</h4>
+    <p>
+        If you want to add support for your tablet on your own, feel free to look at a similar configuration on our GitHub and our configuration documentation.<br/>
+        If you would like help with supporting your tablet, or would prefer us to do it join our <a href="/Discord">Discord</a> and toss a message in <code>#configs</code> or a support channel.
     </p>
-    <ol>
-        <li>
-            Uninstall any tablet drivers that are currently installed.<br/>
-            <em class="ms-3 text-danger">
-                Deleting files is not a proper method to uninstall drivers,
-                <strong>it will cause issues</strong>.
-                Use the <strong>Settings</strong> app or <strong>Control Panel</strong> instead.
-            </em>
-        </li>
-        <li>
-            Use the <a href="https://github.com/X9VoiD/WinUSBCleanup/releases/latest">WinUSBCleanup</a>
-            script to automatically clean up any leftovers that uninstallers failed to clean up.
-        </li>
-    </ol>
 </div>
 
 <hr/>
 
-<h3 class="wiki-nav-item pb-2" id="ghost-cursor">
-    Teleporting cursor position
+<h3 class="wiki-nav-item pb-2" id="tablet-not-functioning-correctly">
+    My tablet is detected, but my settings don't work.
 </h3>
 <div class="ms-3">
     <p>
-        This occurs because another tablet driver is sending input. To resolve this, 
-        uninstall all other tablet drivers on your computer, then replug your tablet.
+        If your area/bindings not working correctly, try following our troubleshooting steps below.<br/><br/>
+
+        Make sure that you have saved your settings by pressing the save button in the bottom right of the OpenTabletDriver interface.<br/><br/>
+
+        If you have done the above and your settings still do not apply then make sure that you don't have other drivers installed on your computer,
+        you can remove these by running and following the instructions for <a href="https://github.com/X9VoiD/TabletDriverCleanup/releases/latest">TabletDriverCleanup</a>,
+        come back here.<br/><br/>
```

Comment:
```suggestion
        then coming back here.<br/><br/>
```

Reply:
This breaks the English and the flow of the sentence unless I'm just stupid?

Reply:
True, the HTML confused me a bit. I'd just drop the "come back here" bit entirely then, it's kinda redundant.